applications: nrf_desktop: Allow using GPIO1 in wheel (#14168)
nrfconnect/sdk-nrf: applications/nrf_desktop/doc/wheel.rst

Enable the module with the :ref:`CONFIG_DESKTOP_WHEEL_ENABLE <config_desktop_app_options>` Kconfig option.

For detecting rotation, the wheel module uses Zephyr's QDEC driver.

> The module can be enabled only when QDEC is configured in DTS and Zephyr's QDEC driver is enabled in Kconfig (that is, :kconfig:option:`CONFIG_QDEC_NRFX` is set).

Reviewer: ```suggestion
You can enable the module only when QDEC is configured in DTS and the Zephyr's QDEC driver is enabled with the :kconfig:option:`CONFIG_QDEC_NRFX` Kconfig option.
```
